### PR TITLE
Newsletters: add 319 (2024-09-06)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Merge summaries @Gustavojfe

### CODE REVIEW

**Comment by harding** [comment]
@Gustavojfe I wrote the summary for the CMake change since it was both simple and a bit different than our usual summaries given all the follow ups.

File: _posts/en/newsletters/2024-09-06-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  [posted][merli stratumfees] to Delving Bitcoin about an extension to
+  [Stratum v2][topic pooled mining] that will allow tracking the amount
+  of fees included in _shares_ when the shares contain transactions
+  selected by an individual miner.  This can be used to adjust the
+  amount paid to the miner by the pool, with miners selecting higher
+  feerate transactions being paid more.
+
+  Merli links to a [paper][merli paper] he co-authored that examines
+  some of the challenges of paying different miners different amounts
+  based on the transactions they select, and which suggests a scheme
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-09-06-newsletter.md:24]:
```suggestion
  based on the transactions they select. The paper suggests a scheme
```
Initially, I was going to suggest replacing "which" with "that" (which I think would be a slight improvement), but this is a pretty long and complex sentence, so maybe break it into two.

File: _posts/en/newsletters/2024-09-06-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  particular, he saw "no justification for the proposed invalidation of
+  64 byte transactions" based on his argument that there's no
+  performance improvement to full nodes in protecting against [merkle
+  tree vulnerabilities][topic merkle tree vulnerabilities] like
+  CVE-2012-2459 by forbidding 64-byte transactions in comparison to
+  other checks that can be performed without a consensus change (and,
+  indeed, those checks are being performed now).  The consensus cleanup
+  proposal champion, Antoine Poinsot, appeared to [agree][poinsot cache]
+  about this full node aspect: "The advantage I initially mentioned
+  about how making 64-bytes transactions invalid could help caching
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-09-06-newsletter.md:53]: Inside a quote so maybe not, but clearly just a typo (or maybe add "[sic]" but I don't love that unless it's formal writing such as an academic paper):
```suggestion
  about how making 64-bytes transactions invalid could help catching
```

**Comment by Gustavojfe** [comment]
just pushed commit to add all merge summaries.

File: _posts/en/newsletters/2024-09-06-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  uneconomical outputs] calculations when checking counterparty feerates, to
+  avoid unnecessary force closures caused by sudden fee spikes. This PR also
+  splits `ConfirmationTarget::OnChainSweep` into `UrgentOnChainSweep` and
+  `NonUrgentOnChainSweep` to distinguish between time-sensitive (e.g., with
+  expiring [HTLCs][topic htlc]) and non-urgent force closures.
+
+- [HWI #742][] adds support for the Trezor Safe 5 hardware signing device.
+
+- [BIPs #1657][] adds a new standard field to PSBT outputs for [DNSSEC][dnssec]
+  proofs on [BIP353][]. External devices such as hardware signers may examine
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-09-06-newsletter.md:171]: "on" seemed a little ambiguous here to me maybe:
```suggestion
  proofs when using [BIP353][]. External devices such as hardware signers may examine
```

File: _posts/en/newsletters/2024-09-06-newsletter.md (hunk 2 above)
Comment (harding) [reply _posts/en/newsletters/2024-09-06-newsletter.md:53]: Leaving as-is.  I believe "caching" is the correct term.  He's talking about storing hashes of consensus invalid blocks in a cache.

File: _posts/en/newsletters/2024-09-06-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+  based on the transactions they select. The paper suggests a scheme
+  that is compatible with a _pay per last N shares_ (PPLNS) pooled
+  mining payout scheme.  His post links to two in-progress
+  implementations of the scheme.
+
+- **OP_CAT research fund:** Victor Kolobov [posted][kolobov cat] to the
+  Bitcoin-Dev mailing list to announce a $1 million fund for research
+  into a proposed soft fork to add an [OP_CAT][topic op_cat] opcode.
+  "Topics of interest include but are not limited to: the security
+  implications of OP_CAT activation on Bitcoin, `OP_CAT`-based computing
```
Comment (jirijakes) [inline _posts/en/newsletters/2024-09-06-newsletter.md:40]: For consistency in this paragraph:

```suggestion
  implications of `OP_CAT` activation on Bitcoin, `OP_CAT`-based computing
```

**Comment by bitschmidty** [comment]
Made a few small formatting fixups. Squashed and merged. Thank you @harding @Gustavojfe @LarryRuane @jirijakes for contributing this week! 🚀

**Pull request merged**